stat/test/wilcoxon_hyp_test.go

```diff
@@ -6,45 +6,39 @@ package test
 
 import (
 	"fmt"
+	"strconv"
 	"strings"
 	"testing"
 )
 
 func TestWilcoxonSignedRankTest(t *testing.T) {
```

Comment:
We always (well, almost always) add `t.Parallel()` at the beginning of a unit test.